Project fails to compile

Seems like even a basic example fails to compile for me, likely due to the constraint on the `azure-pipeline-go` library.

- Sample Gist: https://gist.github.com/ehotinger/9589082621a692aba6668ea820ea1c6a
- Run `dep init`
- Run `dep ensure`
- Run `go build -v`

> vendor/github.com/Azure/azure-storage-blob-go/2016-05-31/azblob/credential_anonymous.go:15:21: undefined: pipeline.FactoryFunc

`$ dep status`
> PROJECT                                 CONSTRAINT  VERSION  REVISION  LATEST  PKGS USED
> github.com/Azure/azure-pipeline-go      1.1.3       1.1.3    f4da77e   1.1.3   1
> github.com/Azure/azure-storage-blob-go  ^0.1.2      0.1.2    c91e48a   0.1.2   1

Also tested a build directly against `master` - same issue.

---

EDIT: this is because of the 1.1.3 constraint put in the project for `azure-pipeline-go`. It needs to be 1.1.4; the upstream source is out of sync.

The toml file might be bad. Use go get and let me know if you have problems.

-- Jeffrey Richter (watch Architecting Distributed Cloud Apps<https://aka.ms/RichterCloudApps> on YouTube<https://aka.ms/RichterCloudApps> or edX<https://aka.ms/edx-devops200_9x-about>)
________________________________
From: Eric Hotinger <notifications@github.com>
Sent: Friday, February 2, 2018 12:43 PM
To: Azure/azure-storage-blob-go
Cc: Subscribed
Subject: [Azure/azure-storage-blob-go] Project fails to compile (#20)


Seems like even a basic example fails to compile for me, likely due to the constraint on the azure-pipeline-go library.

  *   Sample Gist: https://gist.github.com/ehotinger/9589082621a692aba6668ea820ea1c6a
  *   Run dep init
  *   Run dep ensure
  *   Run go build -v

vendor/github.com/Azure/azure-storage-blob-go/2016-05-31/azblob/credential_anonymous.go:15:21: undefined: pipeline.FactoryFunc

$ dep status

PROJECT CONSTRAINT VERSION REVISION LATEST PKGS USED
github.com/Azure/azure-pipeline-go 1.1.3 1.1.3 f4da77e 1.1.3 1
github.com/Azure/azure-storage-blob-go ^0.1.2 0.1.2 c91e48a<https://github.com/Azure/azure-storage-blob-go/commit/c91e48a7b69c952a7699a66e4a7de825945b6b9f> 0.1.2 1

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-storage-blob-go/issues/20>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ACK0vy1kPRY797Xrm9AtpdzCtFVAVbH7ks5tQ3OEgaJpZM4R3wMs>.

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -4,8 +4,8 @@
 [[projects]]
   name = "github.com/Azure/azure-pipeline-go"
   packages = ["pipeline"]
-  revision = "f4da77e3846319876aebb41f2e6dccc6b1d7f1e0"
-  version = "1.1.3"
+  revision = "9804b770d6810999e2bcb7480fb509504e72b2ba"
+  version = "0.1.4"
 
 [[projects]]
   branch = "v1"
@@ -16,6 +16,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "0a0067b54adaf72c0e68dfb59225c7879de5863ff0c951e115e009ad3ecbe7bf"
+  inputs-digest = "31410b00fb3923035bcfdfb37ad828c04e176bd86f7367b5f8df9548e5cc93a2"
   solver-name = "gps-cdcl"
   solver-version = 1

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -1,7 +1,7 @@
 # Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
 [[constraint]]
   name = "github.com/Azure/azure-pipeline-go"
-  version = "1.1.3"
+  branch = "master"
 
 [[constraint]]
   name = "gopkg.in/check.v1"  # This project is only needed if you run "go test" on the Azure Storage packages

